### PR TITLE
Rk 1 network and pipeline

### PR DESCRIPTION
**Pull request by RaphaelKreft**
New files this PR will add: package jpldd/ and dataset/minidepth and scripts/run_inference_jpldd
Other files: edited because of issues in these files regarding my current implementation

### CODE REVIEW

File: gluefactory/datasets/image_folder.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _init(self, conf):
                     raise ValueError(
                         f"Could not find any image in folder: {conf.images}."
                     )
-                self.images = [i.relative_to(conf.images) for i in self.images]
+                self.images = [i.relative_to(conf.images) for i in self.images]  #Todo: relative paths lead to FileNotFoundError
```
Comment (FelixMoeller3) [inline gluefactory/datasets/image_folder.py]: Do you have a suggestions of how we could fix this, i.e. is there any issue with just using the absolute path? That might also make the correction in __getitem__ redundant

File: gluefactory/scripts/run_training_jpldd.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
+"""
```
Comment (FelixMoeller3) [inline gluefactory/scripts/run_training_jpldd.py]: Do we still need this file?

**Review by FelixMoeller3 — APPROVED** [review]
Looks good to me, but let's talk about the two comments I made. No major discussion points but since we are merging into main I would question every change we make

File: gluefactory/datasets/image_folder.py (hunk 1 above)
Comment (RaphaelKreft) [reply gluefactory/datasets/image_folder.py]: image_folder.py is a file we dont use, I just used it as inspiration to implement the dataset. The todo is not valid anymore and I will remove it. No FileNotFound Errors occur. 

FYI: We use relative image paths as they are the part that distinguisehs images from each other. together with conf.images and DATA_PATH we can easily reconstruct the absolute path.

File: gluefactory/scripts/run_training_jpldd.py (hunk 2 above)
Comment (RaphaelKreft) [reply gluefactory/scripts/run_training_jpldd.py]: Not necessarily, it was just a playground :) Could keep it it case we need to run it outside train.py... But not needed really

**Comment by RaphaelKreft** [comment]
I will revise this PR and cleanup the code before merging.

**Pull request merged**